pkg/agent/datamodel/types.go

```diff
@@ -1427,6 +1427,7 @@ type NodeBootstrappingConfiguration struct {
 	KubeletClientTLSBootstrapToken *string
 	FIPSEnabled                    bool
 	HTTPProxyConfig                *HTTPProxyConfig
+	KubeletConfig                  map[string]string
```

Comment:
can we remove KubeletConfig from type KubernetesConfig struct ? I think you will find some more places referencing this after doing that